app/models/trainee.rb

```diff
@@ -1,4 +1,6 @@
 class Trainee < ApplicationRecord
+  enum record_type: { assessment_only: 0, other: 1 }
```

Comment:
Do we need to add this yet seeing as it's not possible to create an 'other' route?

Reply:
removed `Other` option as there will be no recorded with this value. As we develop move routes we can add the route name to this enum

Done in 4edd2d9